subtests/docker_cli/namespaces/namespaces.py

```diff
@@ -0,0 +1,71 @@
+"""
+Summary
+---------
+
+Tests relating to namespaces. As of 2016-08-23 only mount namespaces
+but it's likely we'll need tests for user namespaces.
+
+Operational Summary
+----------------------
+
+#. See individual tests.
+"""
+
+from autotest.client import utils
+from dockertest import subtest
+from dockertest.dockercmd import AsyncDockerCmd
+from dockertest.images import DockerImage
+from dockertest.subtest import SubSubtest
+
+
+class namespaces(subtest.SubSubtestCaller):
+    pass
+
+
+class namespaces_base(SubSubtest):
+    pass
```

Comment:
Yeah, your fix is what I suggested! Excellent mind-reading :grinning: 

There shouldn't be many/any base-classes that are empty.  There are reasons for sub-subtests to be empty though- Normally because they're only difference is in configuration option values.  If there are subtests with empty base-classes I'd consider that a style-bug.

This is all mainly because of my current bias against extraneous code, given how badly I bit myself in the butt with ADEPT.  I even suggested a few in API additions you've made in the past, and you (rightly so) shot me down with: "If it's __actually__ needed at some future point, then it can be added.  For now, best to keep it simple."